theseus_gui/src/pages/instance/Mods.vue

```diff
@@ -173,6 +177,10 @@ function updateSort(projects, sort) {
       })
   }
 }
+
+const searchMod = () => {
+  router.push({ path: '/browse', query: { projectType: 'mod', instance: route.params.id } })
+}
```

Comment:
IMO this should be managed in state (like selectedInstance) or something. if a user exits the flow then the selectedInstance should be set to null